data/yam/autoyast/create_hdd_gnome_x86_64_RAID_encrypted_home.xml

```diff
@@ -0,0 +1,534 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <add-on t="map">
+    <add_on_products t="list">
+      <listentry t="map">
+        <media_url>https://updates.suse.com/SUSE/Products/SLE-Module-Basesystem/15-SP5/x86_64/product?uqBLT_zaCrKaPl66l5TOK5kz1Cm-wowzT03E-BxbbNN61C48hWiJ1-7NbkMAlNs-qLDTweuinZKLKOYb-KmIOI3X8uHyFpDJxBf_VePKxTpWdGbNYcOUum6IP-OykxLq-MU_z3vNOIl_Hephpy_xvA9myis34Y5X</media_url>
+        <product>SLE-Module-Basesystem15-SP5-Pool</product>
+        <product_dir/>
+      </listentry>
+      <listentry t="map">
+        <media_url>https://updates.suse.com/SUSE/Products/SLE-Module-Server-Applications/15-SP5/x86_64/product?qKA5x9nh2ee-G2X0H6XSIVLXdT8cH8US1qQJyo9D1eOsLMBqNpY1R7-Qj_5BDQCidvaENJM9lKo7kKVXJm6QUyvdcxYrzLvAB-f-FUSjEFIF0aoWptANPknd0DIc5BrszSXm_foNwr3Q5n_Q-YxzvLcwFtax-XYMfia6jWrvUWK5</media_url>
+        <product>SLE-Module-Server-Applications15-SP5-Pool</product>
+        <product_dir/>
+      </listentry>
+      <listentry t="map">
+        <media_url>https://updates.suse.com/SUSE/Products/SLE-Module-Python3/15-SP5/x86_64/product?4SZ0rt6mKcY6w2TlOlBq3LY4mqjxfU0YIDqjk9z8DB4h0EKIdhBCiqWLRFUHSMFNCaYTGvzCdejnr2zOGCEdN_wgZcZOMmE3pnF04GE6MXUI16Jb7WmLKKne4XcEPvhvYOQqyA1Xk2f_lYXMB4yh7mkJWyzj</media_url>
+        <product>SLE-Module-Python3-15-SP5-Pool</product>
+        <product_dir/>
+      </listentry>
+    </add_on_products>
+  </add-on>
+  <bootloader t="map">
```

Comment:
could you please compare with autoyast_mini profile in YaST job group, most of this parameters are set by AutoYaST without written them, bootloader, firewall, groups, hosts, perhaps even kdump but not sure, you have to compare running the mini with this product.

Reply:
Updated profile to remove the t="map".

Reply:
that is correct syntax, was introduced later in AutoYaST, you should not remove it, please keep it (otherwise we can find strange error in some point).  I'm talking about full blocks.

Reply:
Recovered the 't="map"', compact bootloader, removed firewall,groups,hosts, and kdump. Works fine: https://openqa.suse.de/tests/14728910#